src/installer/install.ts

```diff
@@ -61,23 +61,37 @@ export async function downloadAndInstall(file: ClientFile): Promise<string> {
     let clientExecutableTmpPath;
     if (extractedDir) {
         const executable = getExecutable(file);
-        const execuatbleFileGlob = `${extractedDir}/**/${executable}`;
+
+        // array consisting of possiblilties of executable file name
+        const execuatbleFileGlobArray: string[] = [];
```

Comment:
@tetchel since we are getting many possibilities of executable file format, so adding this in the array to simpliy the things.